src/components/footer/footer.styles.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import { ReactComponent as Logo } from '../../assets/logo.svg';
+
+export const NavContainer = styled.nav`
+  height: 125px;
+  margin-left: 250px;
+  margin-right: 250px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
`space-between` doesn't work here very well. That's probably why you need `padding-left: 7px` in the component below. Using `flex: 1` on the left and right links would be a better option. You can find more info [here](https://profy.dev/project/reddit-timer/review-tips/no-space-between-in-footer)